app/javascript/cp/fetch.js

```diff
@@ -44,6 +44,7 @@ function fetchCheckHelper(URL, body) {
         },
         method: 'POST',
         body: JSON.stringify(body),
+        credentials: 'include',
```

Comment:
sure would be nice to refactor this.. so many places! I'm not sure how.

Reply:
oh, and then you would have a place to comment why it's necessary! for now, comment the first occurence